Assets/Plugins/iOS/Shopify/Cart+ApplePay.swift

```diff
@@ -79,16 +79,27 @@ import PassKit
     
     private static func extractItems(from jsonString: String) -> [JSON]? {
         
-        let dataKey = "Items"
-        
         guard
-            let data      = jsonString.data(using: .utf8),
-            let itemsJson = try? JSONSerialization.jsonObject(with: data) as? JSON,
-            let items     = itemsJson?[dataKey] as? [JSON]
-        else {
-            return nil
+            let data = jsonString.data(using: .utf8),
+            let object = try? JSONSerialization.jsonObject(with: data),
+            let stringCollection = object as? [String]
+            else {
+                return nil
+        }
+        
+        var itemCollection = [JSON]()
+        for string in stringCollection {
```

Comment:
Nit: Instead of using for/loop and another list you can use `flatMap`:

```swift
return stringCollection.flatMap { string in
    if let stringData  = string.data(using: .utf8), let itemObject = try? JSONSerialization.jsonObject(with: stringData), let itemJson   = itemObject as? JSON {                 
        itemCollection.append(itemJson)
    } else {
        return nil
    }
}
```